PWGGA/GammaConv/AliCaloPhotonCuts.cxx

```diff
@@ -1498,6 +1509,10 @@ void AliCaloPhotonCuts::InitializeEMCAL(AliVEvent *event){
     fGeomEMCAL              = AliEMCALGeometry::GetInstance();
     if(!fGeomEMCAL){ AliFatal("EMCal geometry not initialized!");}
 
+    //retrieve pointer to CaloIsolation Instance
+    fCaloIsolation = (AliPhotonIsolation*)AliAnalysisManager::GetAnalysisManager()->GetTask(fCaloIsolationName.Data());
```

Comment:
Could you please make this check only if isolation is actually required? Otherwise this will run into AliFatal for everyone else.

Reply:
I mean only if fPhotonIsolation is kTRUE.